public/css/themes/solarized-dark.less

```diff
@@ -266,3 +317,65 @@ textarea {
     color: @icinga-blue;
   }
 }
+
+/* Form styles */
+
+@input-background: lighten(@base02, 5);
+
+.icinga-form,
+.controls {
```

Comment:
It's `.icinga-controls` and since you're only changing the style here `.icinga-form` is not required.